docs/getting-started/getting-started-with-globus.md

```diff
@@ -0,0 +1,33 @@
+# User Guide: Transferring Data to RCIF_CHPC using Globus
+
+## Globus Overview
+
+Globus is a secure, reliable data transfer service designed for research and scientific computing. It allows users to move large datasets between institutions, labs, and computing facilities.
+
+## Using Globus Personal Connect
+
+1. Download and install Globus Personal Connect on your local machine from:
+   https://www.globus.org/globus-connect-personal
+2. Set up an endpoint on your computer to enable file transfers.
+
+## Transferring Data to RCIF_CHPC
+
+1. Log in to the Globus web interface (https://app.globus.org).
+2. In the File Manager, add the RCIF_CHPC Collection as a destination:
+   - Collection name: RCIF_CHPC Collection (/ceph)
+   - Endpoint: RCIF_CHPC
+3. Select your local endpoint as the source.
+4. Choose files/folders to transfer and initiate the transfer.
+
+## Key Information
+
+- Endpoint: RCIF_CHPC
+- UUID: 4e6887fc-b5da-4283-93cb-dbfb7e2409d2
+- URL: https://app.globus.org/file-manager/collections/4e6887fc-b5da-4283-93cb-dbfb7e2409d2/overview
+
+## For Assistance
+
+- Contact: support@rcif.wustl.edu
+- Additional info: https://docs.chpc.wustl.edu
```

Comment:
Please remove this - we only maintain relative links and this would be self-referencing so unnecessary